library/std/src/backtrace.rs

```diff
@@ -147,11 +147,15 @@ fn _assert_send_sync() {
     _assert::<Backtrace>();
 }
 
-struct BacktraceFrame {
+/// A single frame of a backtrace.
+#[derive(Debug)]
```

Comment:
I think we’ll probably want to implement `Debug` manually for this type to match the format produced by `Backtrace`.

Reply:
Is that within scope for this PR? Or should that perhaps go in a separate one?

Reply:
I think it would be within scope, since we're making this type public now.